components/script/script_thread.rs

```diff
@@ -181,6 +185,8 @@ impl InProgressLoad {
             is_visible: true,
             url: url,
             origin: origin,
+            navigation_start: 0,
+            navigation_start_precise: 0.,
```

Comment:
Why don't we store the values here? No need to for any of the constellation to be aware of the timing code at all.